### ISSUE TITLE
BetterBackpacks don't update content data

### ISSUE THREAD

**Issue by pinocchiowasjew** [Aroma1997/BetterChests#106]
When I make a BetterBackpack, the items inside it are stored without any problems. However, after some time, whenever I retrieve an item from -- or store an item to -- any backpack and close its GUI and reopen it, its contents remain the same as from before modifying them. The only difference is that I end up either missing items from my inventory or having duplicates. This applies to upgrades and whitelists as well as items. I am using the latest versions of BetterChests and Aroma1997Core.

The log doesn't show anything related to this. Thus, here's a [.gif](https://gyazo.com/b173bd1045422d203a23932e28422c99)

**Comment by Aroma1997** [comment]
Could you still send me a log file of when you put items into a backpack and they vanish?

**Comment by pinocchiowasjew** [comment]
Yes. Here it is (bug occurred at 9:45 PM):
["latest"](https://drive.google.com/file/d/1OtD38vegrpb99Y41pbXZf7sPkji_n_nm/view?usp=sharing)
["fml-client-latest"](https://drive.google.com/file/d/1OUJtoWW9wzo0K8EYU9Ihui_l0tfJ-ERZ/view?usp=sharing)

<details> <summary> </summary>Parenthetically, while trying to get the aforementioned bug to occur, a new one arose:
After dropping a BetterBackpack, picking it up and opening it, my client crashed.
[Report](https://drive.google.com/file/d/1m1xF9ZLlanAv9jekb0Ihug7ZCUS61zAH/view?usp=sharing)
</details>

**Comment by Aroma1997** [comment]
Does this happen every time or only sometimes?

**Comment by pinocchiowasjew** [comment]
From what I've experienced so far, it happens at a random point of a session. When I load my world, everything seems to be fine but goes wrong after some time (1+ hours?).  When that happens, the bug persists until I unload the world and load it again.

After going through the FML log, I noticed that Sponge appeared several times next to MalisisDoors and, as I run a singleplayer world, that seemed odd to me. I disabled MalisisDoors and MalisisCore and tested it. The result is the same.

**Comment by Aroma1997** [comment]
Can you try uninstalling sponge and tell me whether the bug still persists?

**Comment by pinocchiowasjew** [comment]
As far as I know, Sponge is not installed as I didn't install it to begin with. I believe that I can't do anything in this case.

**Comment by Aroma1997** [comment]
I'm sorry about that, I read the log file incorrectly. Sponge isn't installed...
Since I can't reproduce the issue, it's a bit hard to fix it. Could you send me your modpack + world (In a Private message, if you don't want to share it publicly), so I can reproduce the issue?

**Comment by pinocchiowasjew** [comment]
Yes. [Here it is](https://drive.google.com/file/d/10RV7A5a7SyDerGmdfjkXqxmcq94KC6Rb/view?usp=sharing).

**Comment by Aroma1997** [comment]
Ok, I couldn't reproduce the issue. Any idea on how to reproduce this?

**Comment by pinocchiowasjew** [comment]
Yes. I play for at least about an hour before it happens; standing works but seems to take longer (game has to not be paused). Interacting with the world seems to make the bug occur faster, though I cannot confirm this.

**Comment by Aroma1997** [comment]
How are you opening the bag? by right-clicking it? by using the keybind? Does it matter whether you have upgrades in the bag?

**Comment by pinocchiowasjew** [comment]
I open it by right-clicking it. I cannot use a keybind as the game crashes. The bug occurs regardless of whether the bag has upgrades or not.
